src/py/flwr/server/superlink/ffs/ffs_factory.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Factory class that creates Ffs instances."""
+
+
+from logging import DEBUG
+from typing import Optional
+
+from flwr.common.logger import log
+
+from .disk_ffs import DiskFfs
+from .ffs import Ffs
+
+
+class FfsFactory:
```

Comment:
Do we use this in any of the follow-up PRs? If not, we can remove it.

Reply:
It is used in the final PR, but I can separate it to another PR if you prefer

Reply:
yes, that'd be great

Reply:
@danieljanes Done!